code/modules/clothing/factions/vysoka.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/clothing/suit/storage/vysoka_m
+	name = "chokha"
+	desc = " A woolen coat with a high neck commonly worn on Vysoka."
+	desc_fluff = "Chokhas are a common choice of outerwear for men and women on Vysoka, and are often colored differently based on their city - or host - of origin. The chokha has become something of a cultural symbol of Vysoka, and is often worn off-world by its citizens."
+	icon = 'icons/clothing/suits/coats/vysoka_chokha_male.dmi'
+	icon_state = "chokha_vysoka_m"
+	item_state = "chokha_vysoka_m"
+	contained_sprite = TRUE
+
+/obj/item/clothing/suit/storage/vysoka_f
+	name = "chokha"
+	desc = " A woolen coat with a high neck commonly worn on Vysoka."
+	desc_fluff = "Chokhas are a common choice of outerwear for men and women on Vysoka, and are often colored differently based on their city - or host - of origin. The chokha has become something of a cultural symbol of Vysoka, and is often worn off-world by its citizens."
+	icon = 'icons/clothing/suits/coats/vysoka_chokha_female.dmi'
+	icon_state = "chokha_vysoka_f"
+	item_state = "chokha_vysoka_f"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/soft/vysoka
+	icon_state = "cap_vysoka"
+	item_state = "cap_vysoka"
+	icon = 'icons/clothing/head/vysoka_cap.dmi'
+	name = "vysokan fur cap"
```

Comment:
name
desc
icon
the rest

use the same format as the others bleez

Reply:
OK boss will do soon